afs/afs-ws/afs-ws-storage/src/main/java/com/powsybl/afs/ws/storage/NodeEventClient.java

```diff
@@ -54,6 +70,57 @@ public void onError(Throwable t) {
 
     @OnClose
     public void onClose(Session session) {
-        LOGGER.trace("Node event websocket session '{}' closed for file system '{}'", session.getId(), fileSystemName);
+        int counter = 0;
+        boolean toReconnection = true;
+        long dateInitial = Instant.now().getEpochSecond();
+        while (toReconnection) {
+
+            long dateReconnection = Instant.now().getEpochSecond();
+            if (dateReconnection - dateInitial > config.getReconnectionMax()) {
+                LOGGER.warn("Node event websocket session '{}' closed for file system '{}'", session.getId(), fileSystemName);
+                return;
+            }
+            updateReconnectionInterval(counter);
+            try {
+                URI wsUri = RemoteListenableAppStorage.getWebSocketUri(config.getRestUri());
+                URI endPointUri = URI.create(wsUri + "/messages/" + AfsRestApi.RESOURCE_ROOT + "/" +
+                        AfsRestApi.VERSION + "/node_events/" + fileSystemName);
+                LOGGER.debug("Connecting to node event websocket at {}", endPointUri);
+                WebSocketContainer container = ContainerProvider.getWebSocketContainer();
+                container.connectToServer(this, endPointUri);
+                toReconnection = false;
+
+            } catch (DeploymentException e) {
+                if (e.getCause().getCause() instanceof ConnectException) {
+                    try {
+                        LOGGER.error(e.getCause().getCause().getMessage(), e);
```

Comment:
this should be a warning without stack trace. The warning message should mention when the next reconnection attempt will be made

Reply:
It's done